modin/pandas/test/test_groupby.py

```diff
@@ -1968,6 +1971,26 @@ def compare(obj1, obj2):
     compare(reference_by, result_by)
 
 
+def test_groupby_with_virtual_partitions():
+    # from https://github.com/modin-project/modin/issues/4464
+    md_df = pd.DataFrame(test_data["int_data"])
+    pd_df = pandas.DataFrame(test_data["int_data"])
```

Comment:
```suggestion
    md_df, pd_df  = create_test_dfs(test_data["int_data"])
```

I just remembered that we have this nice utility.